### ISSUE TITLE
Dead links

### ISSUE THREAD

**Issue by vidartf** [hyperspy/hyperspy#1464]
There are some dead links around:
```
C:\dev\hyperspy [RELEASE_next_minor ≡ ]> pip install pytest-check-links
C:\dev\hyperspy [RELEASE_next_minor ≡ ]> py.test --check-links -k .html
============================= test session starts =============================
platform win32 -- Python 3.5.2, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: C:\dev\hyperspy, inifile: pytest.ini
plugins: nbval-0.4.1, tornado-0.4.5, timeout-1.2.0, cov-2.4.0, check-links-0.0.2
collected 1643 items

doc\_static\articles_using_hyperspy.html .....F..F.FF...
hyperspy\tests\drawing\test_plot_image.ipynb .

================================== FAILURES ===================================
 C:\dev\hyperspy\doc\_static\articles_using_hyperspy.html: http://link.aip.org/link/?APL/93/112505/1
http://link.aip.org/link/?APL/93/112505/1: <urlopen error [Errno 11001] getaddrinfo failed>
 C:\dev\hyperspy\doc\_static\articles_using_hyperspy.html: 16/j.ultramic.2010.10.001
16/j.ultramic.2010.10.001: No such file: C:\dev\hyperspy\doc\_static\16\j.ultramic.2010.10.001
 C:\dev\hyperspy\doc\_static\articles_using_hyperspy.html: http://dx.doi.org/10.1088/0957-4484/21/14/145606
http://dx.doi.org/10.1088/0957-4484/21/14/145606: HTTP Error 403: Forbidden
 C:\dev\hyperspy\doc\_static\articles_using_hyperspy.html: http://iopscience.iop.org/0957-4484/21/14/145606/
http://iopscience.iop.org/0957-4484/21/14/145606/: HTTP Error 403: Forbidden
============================ 1627 tests deselected ============================
============ 4 failed, 12 passed, 1627 deselected in 29.03 seconds ============
```

Sphinx also has a linkcheck command:
```
cd docs
make linkcheck   (alternatively makefile.bat linkcheck on Windows)
```
gives a multitude of broken links and permanent redirects.

**Comment by francisco-dlp** [comment]
Thanks!

**Comment by vidartf** [comment]
Closed in #1475
